precode.go

```diff
@@ -13,29 +14,55 @@ import (
 // сгенерированных чисел.
 func Generator(ctx context.Context, ch chan<- int64, fn func(int64)) {
 	// 1. Функция Generator
-	// ...
+	// Начальное значение
+	var v int64 = 1
+
+	for {
+		select {
+		case <-ctx.Done():
+			close(ch)
+			return
+		default:
+			ch <- v
+			fn(v)
+			v++
+		}
+	}
 }
 
 // Worker читает число из канала in и пишет его в канал out.
 func Worker(in <-chan int64, out chan<- int64) {
 	// 2. Функция Worker
-	// ...
+	defer close(out)
+	for {
+
+		v, ok := <-in
+		if !ok {
+			break
+		}
+		out <- v
+		time.Sleep(1 * time.Millisecond)
+	}
+
 }
 
 func main() {
 	chIn := make(chan int64)
 
 	// 3. Создание контекста
-	// ...
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Лучше так делать
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
	defer cancel()
```

Потому что у тебя выход из функции (или паника) может случиться раньше чем через 1 секунду, и дефер сработает, а твой код нет